crates/stackable-operator/src/builder/pod/mod.rs

```diff
@@ -533,7 +573,11 @@ impl PodBuilder {
             }),
             security_context: self.security_context.clone(),
             tolerations: self.tolerations.clone(),
-            volumes: self.volumes.clone(),
+            volumes: if self.volumes.is_empty() {
+                None
+            } else {
+                Some(self.volumes.clone().into_values().collect())
+            },
```

Comment:
suggestion: Please move this outside of the struct creation (same as above).

Reply:
Moved in b7fcdaa